### PR TITLE
domd: domu: add support for core-image-minimal

### PR DESCRIPTION
**Pull request by rshym**
This commit adds possibility to build our components inside core-image-minimal.
Parts that are common for core-image-minimal and core-image-weston are moved into the common_install.inc file.

### CODE REVIEW

**Comment by rshym** [comment]
Tested by the builds of virtio, and standard yamls. For h3-ulcb-4x2g and h3ulcb-4x2g-kf.

File: meta-xt-driver-domain/recipes-graphics/images/common_install.inc
Comment (lorc) [inline meta-xt-driver-domain/recipes-graphics/images/common_install.inc:1]: It is strange to see this in "recipes-graphics" tbh. Is there a way to move it into more appropriate place?

Comment (rshym) [reply meta-xt-driver-domain/recipes-graphics/images/common_install.inc:1]: Done. Files `core-image-minimal.bbappend` and `common_install.inc` are moved to recipes-core.

**Review by lorc — APPROVED** [review]
`Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Comment by rshym** [comment]
`Ack` is added. Tested. ready to be merged.

**Pull request merged**